WordPress/Classes/Services/RecentSitesService.swift

```diff
@@ -37,7 +37,7 @@ class RecentSitesService: NSObject {
     /// Initialize the service using the standard UserDefaults as the database.
     ///
     convenience override init() {
-        self.init(database: UserDefaults() as KeyValueDatabase)
+        self.init(database: UserDefaults.standard as KeyValueDatabase)
```

Comment:
Obj-C complained when I tried to move this to a default value for the `database` param in the `init` call above, so I just kept the change here.